packages/transactional-adapters/transactional-adapter-pg-promise/src/lib/transactional-adapter-pg-promise.ts

```diff
@@ -0,0 +1,35 @@
+import { TransactionalAdapter } from '@nestjs-cls/transactional';
+import { IDatabase, ITask } from 'pg-promise';
+
+export type PgPromiseDbOrTx = IDatabase<unknown> | ITask<unknown>;
+
+export interface PgPromiseTransactionalAdapterOptions {
+    /**
+     * The injection token for the pg-promise instance.
+     */
+    pgPromiseInstanceToken: any;
```

Comment:
This refers to the _database_ instance, not pg-promise instance, so it should be named accordingly.

Reply:
Question: I don't quite understand what this is used for? It's [aliased](https://github.com/Papooch/nestjs-cls/blob/main/packages/transactional/src/lib/plugin-transactional.ts#L20) to `TRANSACTION_CONNECTION`, but that token doesn't seem to be used anywhere else. Unless I missed something?

Reply:
It is used to inject the connection to the adapter options factory [just a few lines below](https://github.com/Papooch/nestjs-cls/blob/8ef1e4061e8aed3be02eb6124c42ac295dff8903/packages/transactional/src/lib/plugin-transactional.ts#L25). It is the injection token for the thing that will be ultimately passed to the `optionsFactory` of `TransactionalAdapter`.

Reply:
Sorry, I don't know how I missed that. Must have been drunk when I looked at your code 🙈